snap/implicit.go

```diff
@@ -43,6 +43,7 @@ var implicitClassicSlots = []string{
 	"opengl",
 	"network-manager",
 	"pulseaudio",
+	"cups-control",
```

Comment:
I know they aren't now and it doesn't have to be in this PR, but it would be nice if these were alphabetized.